backend/app/core/service/loader.py

```diff
@@ -51,17 +107,17 @@ def persist_document_and_metadata(
 
 
 def write_metadata(
-    db: Session,
+    db,
```

Comment:
Why was the type removed?

Reply:
sorry, meant to put that back - done now